Vizzuality/heco-invest
# Chore/change husky

~Use lint-staged with husky to not invoke frontend pre-commit checks when working on backend and vice-versa. Husky advises to have it installed on the root folder, the same for lint-staged (https://github.com/okonet/lint-staged#how-to-use-lint-staged-in-a-multi-package-monorepo).~

Switched to https://github.com/evilmartians/lefthook as it seems to be easier to configure and extend with different hooks. There is still problem with Typescript type check that cannot easily run on staged files, instead it runs on the whole project which takes a lot of time (and that will grow, maybe it should be pre_push hook, not pre_commit?).

## Testing instructions

Make sure to uninstall previously installed husky config `git config --unset core.hooksPath`. Then run `yarn install` at root directory. Do any dummy change that would invoke errors either on the backend or frontend side (or both) and try to add commit.

## Tracking

Not tracked
